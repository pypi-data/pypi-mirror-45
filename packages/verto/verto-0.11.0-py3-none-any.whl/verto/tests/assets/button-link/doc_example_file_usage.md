{button-link link="files/python-sort-example.py" text="Download Python Sorting Example" file="yes"}
