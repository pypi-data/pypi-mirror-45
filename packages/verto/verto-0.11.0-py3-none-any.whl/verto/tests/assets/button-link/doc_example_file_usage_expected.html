<a class="button" href="{% static 'files/python-sort-example.py' %}">Download Python Sorting Example</a>
